packages/repository-modernizer/src/restructure-config.js

```diff
@@ -338,5 +340,152 @@ async function deleteEmptyConfigDir(oldConfigFilePath) {
         oldConfigFileParent = path.dirname(oldConfigFileParent);
     }
 }
+/**
+ *
+ * @param String newConfigFilePath the new config file path in ui.config package
+ *
+ * format config in xml ,cfg,config format to cfg.json
+ */
+async function formatConfig(newConfigFilePath) {
+    let fileName = newConfigFilePath.substring(
+        newConfigFilePath.lastIndexOf("/") + 1
+    );
 
+    if (
+        !fileName.startsWith(
+            "org.apache.sling.jcr.repoinit.RepositoryInitializer"
+        ) &&
+        (newConfigFilePath.endsWith(".xml") ||
+            newConfigFilePath.endsWith(".config") ||
+            newConfigFilePath.endsWith(".cfg"))
+    ) {
+        let fileContent = util.getXMLContentSync(newConfigFilePath);
+        let contentToBeWritten = [];
+        let pushContent = false;
+        let path = "";
+        if (newConfigFilePath.endsWith(".xml")) {
+            path =
```

Comment:
I would advocate to move the content of `if-else` block to separate methods if possible.